### ISSUE TITLE
Create override class for links to be same colour as normal text

### ISSUE THREAD

**Issue by mrmjprice** [wmcadigital/wmn-design-system#188]
Use case: Regular link styling fails accessibility criteria in warning boxes

Solution: Override class so that links are the same colour as normal text
